app/src/main/java/com/example/annakocheshkova/testapplication/manager/exporter/ExporterFactory.java

```diff
@@ -0,0 +1,26 @@
+package com.example.annakocheshkova.testapplication.manager.exporter;
+
+import com.example.annakocheshkova.testapplication.model.Task;
+
+/**
+ * A factory to get the exporter instance we need
+ */
+public class ExporterFactory {
+
+    /**
+     * Enum of all the export types
+     */
+    public enum ExportType {
+        LOCAL_TO_FILE,
+        REMOTE
+    }
+
+    public static Exporter<Task> getTaskExporter(ExportType exportType) {
+        if (exportType == ExportType.LOCAL_TO_FILE) {
+            return new FileExporter<>();
```

Comment:
It seems that this method return a new instance of `FileExporter` no matter what the value of `exportType` is.